### DISCUSSION TITLE
Switch between letters and numbers for alphanumeric words

### DISCUSSION THREAD

**Discussion by lltusharll** [8VIM/8VIM#217] [Ideas]
It would be nice if there's a switch button to switch between numbers and letters quickly. Currently it gets difficult to type some works with digit in it.

**Comment by flide** [comment]
well that button exists ... the left sector is a button, displayed like a
number pad.

**Reply by lltusharll** [reply]
Ya right it exists, just to switch back from digits it feels little far but anyways that's fine.

**Reply by lltusharll** [reply]
It would be nice if we have the spelling correct suggestion as well as next word suggestion.
